coverart.lua

```diff
@@ -173,7 +176,7 @@ local function isValidCoverart(file, icy)
 
     if o.imageExts ~= "" and not imageExts[fileext] then
         msg.debug('"' .. fileext .. '" not in whitelist')
-        return false
+        return nil
```

Comment:
Ok, I remember what I wanted changed now. This logic all seems fine, but I think that the use of `true`, `false` and `nil` for these return values could get confusing to me later when I am trying to maintain the code. I don't necessarily want you to change anything, but I do want you to use variables to represent the different return values so that things are easier to read. For example:

```lua
local VALID_FILE = 1
local INVALID_FILENAME = 2
local INVALID_EXTENSION = 3
```



So this could be changed to:

```suggestion
        return INVALID_EXTENSION
```